axelor-production/src/main/java/com/axelor/apps/production/web/ManufOrderController.java

```diff
@@ -459,4 +460,45 @@ public void computeCostPrice(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void checkMergeValues(ActionRequest request, ActionResponse response) {
+    try {
+      if (request.getContext().get("id") != null) {
+        response.setError(I18n.get("Please select more than one OF"));
+      } else {
+        Object _ids = request.getContext().get("_ids");
+        if (!ObjectUtils.isEmpty(_ids)) {
+          List<Long> ids = (List<Long>) _ids;
+          if (ids.size() < 2) {
+            response.setError(I18n.get("Please select more than one OF"));
+          } else {
+            boolean canMerge = Beans.get(ManufOrderWorkflowService.class).canMerge(ids);
+            if (canMerge) {
+              response.setAlert(I18n.get("Are you sur you want to merge those manuf orders?"));
```

Comment:
typo : sur --> sure
Try not to abbreviate "manufacturing" into "manuf" or at least put a dot : "manuf."
Error messages should be defined in static variables in a separate interface.